webapp/docker-compose.yml

```diff
@@ -5,15 +5,20 @@ services:
     build:
         network: host
         context: ./webapp
-    command: python /webapp/manage.py runserver 0.0.0.0:8000
-    restart: 'always'
+    command: >
+      bash -c "/etc/init.d/cron start &&
```

Comment:
I've had issues in past running cron tabs within docker containers, in particular inheriting the correct env variables - have you tested this compose config on a host? I've had to use something like:

entrypoint: '
  #!/bin/bash
  env >> /etc/environment
  # execute CMD
  echo "$@"
  exec "$@"
'

to have my cron tab actually pick up the correct env vars.

Reply:
Interesting solution. Looks like https://github.com/CogStack/MedCAT/blob/0f0f5bea9cbb51b548d3858654a93b05ce3e111d/webapp/docker-compose.yml#L15 is similar to yours, if ubuntu is used as the host OS.